opennms-doc/guide-admin/src/asciidoc/text/config-files/_template.adoc

```diff
@@ -0,0 +1,23 @@
+
+// REMOVE ME!! Please keep first line an empty line to make sure, the ToC can be build correctly
+==== File: {name-of-the-configuration-file}
```

Comment:
Also, for xml files, there's already a lot of documentation in the schema files (.xsd files).

Couldn't you setup some automation to get some info from these files right here?